app_dart/lib/src/model/proto/internal/scheduler.proto

```diff
@@ -33,6 +33,25 @@ message Target {
     // Name of the scheduler to trigger this target.
     // Defaults to being triggered by cocoon.
     optional SchedulerSystem scheduler = 8 [default = cocoon];
+    optional PhaseConfig presubmit = 9;
+    optional PhaseConfig postsubmit = 10;
+}
+
+// CI Phase (i.e. pre-submit vs post-submit).
+// Next ID: 3
+message PhaseConfig {
+    optional Pool pool = 1;
+    // Allows disabling tests not intended to run in this phase.
+    optional bool enabled = 2;
```

Comment:
in my original design doc, I had both `enabled` and `flaky` here, but it probably makes more sense to have `flaky` at the top level (a target that is flaky post-submit is also flaky pre-submit). I'm assuming we should skip flaky tests pre-submit?

Reply:
Yes. `Target` allows setting `bringup` where it'll handle this logic.

Short term, presubmit will override and not schedule tasks marked `bringup=true`. Long term, it would be good to schedule them but not push checks to collect data.

Reply:
> Yes. `Target` allows setting `bringup` where it'll handle this logic.
> 
> Short term, presubmit will override and not schedule tasks marked `bringup=true`. Long term, it would be good to schedule them but not push checks to collect data.

SGTM.